exercises/practice/darts/tests/Tests.elm

```diff
@@ -0,0 +1,79 @@
+module Tests exposing (tests)
+
+import Darts
+import Expect
+import Test exposing (Test, describe, skip, test)
+
+
+
+-- Return the correct amount earned by a dart's landing position.
```

Comment:
We can remove the comments, the generator leaves them because it's sometimes useful to see when you implement the tests.